skyvern-frontend/src/routes/workflows/editor/nodes/NodeAdderNode/NodeAdderNode.tsx

```
@@ -1,14 +1,46 @@
 import { Handle, NodeProps, Position, useEdges } from "@xyflow/react";
-import type { NodeAdderNode } from "./types";
-import { PlusIcon } from "@radix-ui/react-icons";
+
+import { RadialMenu } from "@/components/RadialMenu";
+import { useIsSkyvernUser } from "@/hooks/useIsSkyvernUser";
+import { useDebugStore } from "@/store/useDebugStore";
 import { useWorkflowPanelStore } from "@/store/WorkflowPanelStore";
 
+import type { NodeAdderNode } from "./types";
+
 function NodeAdderNode({ id, parentId }: NodeProps<NodeAdderNode>) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'NodeAdderNode'. Consider to delete it or rename it.

'NodeAdderNode' is defined here:


(lint/suspicious/noRedeclare)